contributors/design-proposals/customresources-subresources.md

```diff
@@ -82,6 +82,9 @@ type CustomResourceSubResourceScale struct {
     // optional, e.g. “.spec.labelSelector”. Must be under `.spec`.
     // Only JSON paths without the array notation are allowed.
     LabelSelectorPath string `json:"labelSelectorPath,omitempty"`
+    // GVK denotes the gvk of the Scale object sent as the payload
+    // for /scale. It allows transition to future versions easily.
+    GVK schema.GroupVersionKind `json:"gvk,omitempty"`
```

Comment:
Would call it GroupVersionKind, the long version

Reply:
and: Today only `autoscaling/v1.Scale` (is this the official syntax?) is allowed. If left empty, it default to this value.